### ISSUE TITLE
Publish a stable version

### ISSUE THREAD

**Issue by hug-dev** [parallaxsecond/rust-tss-esapi#148]
This top-level issue is to gather all things that need to be done before publishing a stable version, without the `alpha` postfix.
Feel free to edit the list.

- [ ] Only expose and use abstracted Rust-native types (no raw TSS types)
- [x] Add more documentation/example on the API (#21)
- [x] Perform the split with the `-sys` crate, check if the bindings can be committed (#85, #30)
- [x] Ensure all functions return all returned TPM data (#151)

**Comment by gaetanww** [comment]
> Only expose and use abstracted Rust-native types (no raw TSS types)

Would you accept PRs on this? I'm not sure yet whether I'll be able to work on it, but I could have a look at adapting what I've got locally.

**Comment by ionut-arm** [comment]
Of course, we'd be more than happy to have other contributors involved in this!

**Comment by ionut-arm** [comment]
The first bullet-point is not fully covered, but it isn't (wasn't?) a complete blocker, especially for complex types. Closing this,, there will be a new issue looking at the FFI types we should replace
